language-server/modules/langserver-core/src/test/resources/completion/module_const_context/config/config6.json

```diff
@@ -191,7 +192,7 @@
     {
       "label": "varName",
       "kind": "Variable",
-      "detail": "int",
+      "detail": "$CompilationError$",
```

Comment:
According to the spec, `The type of the constant is the intersection of readonly and the singleton type containing just the shape of the value named by the constant.` 

if we take `const int x = 1;` after this fix. The type of x is singleton-type `. 

since the rhs is `unknown type`, we could not create singleton type hence return `compilation error`

Reply:
I see. Please create an issue for this case. LS side should improve this with expected type API

Reply:
@prakanth97 did you create an issue? please link it here.